corehq/apps/domain/management/commands/delete_domain.py

```diff
@@ -41,5 +49,16 @@ def handle(self, domain_name, **options):
                 print("\n\t\tDomain deletion cancelled.")
                 return
         print("Deleting domain {}".format(domain_name))
+
```

Comment:
Could you explain more about why this code is in the management command instead of domain/deletion.py? I want to understand if there's a good reason to break the convention.